scripts/override_respot.py

```diff
@@ -102,15 +102,16 @@ def override_values(tech, year, dr):
         else:
             print("No RES potential techs to override...")
 
-        if snakemake.params.custom_data["elec_demand"]:
-            for country in countries:
-                n.loads_t.p_set.filter(like=country)[buses] = (
-                    (
-                        n.loads_t.p_set.filter(like=country)[buses]
-                        / n.loads_t.p_set.filter(like=country)[buses].sum().sum()
-                    )
-                    * energy_totals.loc[country, "electricity residential"]
-                    * 1e6
-                )
+        """Removed here because this is taken care of in prepare_sector_network"""
+        # if snakemake.params.custom_data["elec_demand"]:
+        #     for country in countries:
+        #         n.loads_t.p_set.filter(like=country)[buses] = (
+        #             (
+        #                 n.loads_t.p_set.filter(like=country)[buses]
+        #                 / n.loads_t.p_set.filter(like=country)[buses].sum().sum()
+        #             )
+        #             * energy_totals.loc[country, "electricity residential"]
+        #             * 1e6
+        #         )
```

Comment:
As mentioned, feel free to remove this fragment completely if you think it's not needed.

As far as I can see, this fragment is commented-out in `prepare_sector_network`. Is it intentional or is there a need to uncomment in `prepare_sector_network`?